sor/src/main/java/com/bazaarvoice/emodb/sor/db/DataWriterDAO.java

```diff
@@ -34,7 +35,7 @@
 
     /** Deletes one or more deltas and replaces them with the specified update. */
     void compact(Table table, String key, UUID compactionKey, Compaction compaction, UUID changeId, Delta delta,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS00107)